docs/faq.rst

```diff
@@ -27,6 +27,7 @@ Array                               Any supported type
 Enumeration                         Any NumPy integer type                          Read/write as integers
 References                          Region and object
 Variable length array               Any supported type                              See :ref:`Special Types <vlen>`
+Any other NumPy dtype               Map to opaque type with specific tag            You must register types manually
 =========================           ============================================    ======================
```

Comment:
I thin the last set of `====` need to span the text in the table, because this is rst.

Reply:
Generally speaking, listtable is a better construct here, but you don't need to change that, Hameer.